src/models/room.js

```diff
@@ -0,0 +1,16 @@
+import DataTypes from 'sequelize';
```

Comment:
The `DataTypes` should be imported using destructuring: `import { DataTypes } from 'sequelize';` to correctly access the data types.